src/reducers/url-state.js

```diff
@@ -306,9 +306,10 @@ export const getVisibleThreadOrder = createSelector(
 );
 export const getURLPredictor = createSelector(
   getURLState,
-  (oldURLState: URLState) => actionOrActionList => {
-    const actionList =
-      'type' in actionOrActionList ? [actionOrActionList] : actionOrActionList;
+  (oldURLState: URLState) => (actionOrActionList: Action | Action[]) => {
+    const actionList: Action[] = Array.isArray(actionOrActionList)
+      ? actionOrActionList
+      : [actionOrActionList];
```

Comment:
Typing more strictly the returned function made it necessary to be stricter about the argument handling.

I don't know how it was typed before...